media/css/firefox/whatsnew/whatsnew-67.scss

```diff
@@ -0,0 +1,500 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+
+@import '../../../protocol/css/includes/lib';
+@import '../../../protocol/css/components/call-out';
+@import '../../../protocol/css/components/feature-card';
+@import '../../../protocol/css/components/hero';
+@import '../../../protocol/css/components/modal';
+@import '../../../protocol/css/components/newsletter-form';
+
+@import '../../protocol/components/state-fxa';
+
+
+main {
+    color: $color-off-black;
+}
+
+//* -------------------------------------------------------------------------- */
+// Up-to-date page header
+
+.c-page-header-up-to-date {
+    @include text-display-xxs;
+    display: none;
+
+    span {
+        background-color: #6EE7C4;
```

Comment:
Should this use a token in the Protocol color pallet? I think we've seen this type of notification in the designs for /thanks, so it seems like something that will appear in multiple places.

Reply:
It's not a colour that 's in the pallet right now 😠 

I do think it's something we should add.